### DISCUSSION TITLE
connecting third-party filters and diff patching

### DISCUSSION THREAD

**Discussion by electrifying** [uBlockOrigin/uBOL-home#242] [Q&A]
Hello, will it be possible to connect a list of third-party filters and diff patching so that filters are updated without waiting for a new version of the application as the authors of the adblock plus extension did?

**Comment by ghost** [comment]
- https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#when-do-filter-lists-update
- https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#is-ubo-lite-a-bad-faith-attempt-at-converting-ubo-to-mv3

**Reply by electrifying** [reply]
> * https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#when-do-filter-lists-update
> * https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#is-ubo-lite-a-bad-faith-attempt-at-converting-ubo-to-mv3

Yes, but what prevents this from being implemented as the authors adblock plus and adguard did?

**Reply by stephenhawk8054** [reply]
They don't go declarative approach like uBOL. It's written in the links above.

Declarative approach is more reliable and efficient but cannot update the filter lists outside extension's updates.
